crates/ra_lsp_server/src/main.rs

```diff
@@ -27,7 +27,7 @@ fn main() -> Result<()> {
 #[derive(Deserialize)]
 #[serde(rename_all = "camelCase")]
 struct InitializationOptions {
-    publish_decorations: bool,
+    highlighting_on: bool,
```

Comment:
Hm, does this mean we can kill [this logic](https://github.com/rust-analyzer/rust-analyzer/blob/c96011833c7017505a9de21f35e5c7dece7446d3/editors/code/src/server.ts#L24-L26) as well?

Reply:
We should be able to kill that logic, but at the moment its working a little bit spookily - the highlighting is being re-enabled when I set the setting back to true, but there is no way `publish_decorations` should be being set back to true. I need to investigate this further.

Reply:
Hang on - that's not right. I'm even more confused to be honest. I'll get back to you on this.